docs/tools/vdb_table/data/qdrant.json

```diff
@@ -158,6 +158,16 @@
     "source_url": "https://qdrant.tech/documentation/guides/quantization/#binary-quantization",
     "comment": "via Binary Quantization of index"
   },
+  "index_types": {
+    "value": [
+      "FLAT",
+      "HNSW",
```

Comment:
I think it makes sense to add HNSW-SQ as well per the docs?
https://qdrant.tech/documentation/guides/quantization/